From af5b354a536bdec1df77db383a29deb8c686d6a5 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Mon, 17 Nov 2014 16:40:06 +0800
Subject: [PATCH] MLK-9877: ARM: dts: imx6sx: add gpu clock for gpc driver

Add gpu clock for pu-enable, otherwise, GPU may crash in some test cases.

Signed-off-by: Robin Gong <b38343@freescale.com>
---
 arch/arm/boot/dts/imx6sx.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index e8df520..781e709 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -768,6 +768,7 @@
 				reg = <0x020dc000 0x4000>;
 				interrupts = <GIC_SPI 89 IRQ_TYPE_LEVEL_HIGH>;
 				fsl,mf-mix-wakeup-irq = <0x7c00000 0x3d00 0x0 0x400200>;
+				clocks = <&clks IMX6SX_CLK_GPU>;
 				#power-domain-cells = <1>;
 			};
 
-- 
1.8.0

